File: src/main/java/phoenix/utils/block/IColoredBlock.java
package phoenix.utils.block;

import net.minecraft.client.renderer.color.IBlockColor;
import net.minecraft.client.renderer.color.IItemColor;
import net.minecraftforge.api.distmarker.Dist;
import net.minecraftforge.api.distmarker.OnlyIn;
/*
 * Это стыбзено из туманного мира
 */
public interface IColoredBlock
{
    @OnlyIn(Dist.CLIENT)
    public IBlockColor getBlockColor();

    @OnlyIn(Dist.CLIENT)
    public IItemColor getItemColor();
}

File: src/main/kotlin/phoenix/api/block/IColoredBlock.kt
package phoenix.api.block

import net.minecraft.client.renderer.color.IBlockColor
import net.minecraft.client.renderer.color.IItemColor
import net.minecraftforge.api.distmarker.Dist
import net.minecraftforge.api.distmarker.OnlyIn

/*
* Это стыбзено из туманного мира
*/
interface IColoredBlock
{
    @OnlyIn(Dist.CLIENT)
    fun getBlockColor(): IBlockColor?

    @OnlyIn(Dist.CLIENT)
    fun getItemColor(): IItemColor?
}